lib/rubocop/cop/rspec/eq.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Use `eq` instead of `be ==` to compare objects.
+      #
+      # @example
+      #   # bad
+      #   expect(foo).to be == true
```

Comment:
I suggest using some other value rather than ‘true’ to avoid any confusion with be_truthy etc.

Reply:
Fixed in https://github.com/rubocop/rubocop-rspec/pull/1706

Reply:
Oh, I had overlooked this comment🙇‍♂ Thanks for catching this.